### PR TITLE
Member feature

### PR DESCRIPTION
**Pull request by tomkelele**
show list members, add and delete member

### CODE REVIEW

File: app/Http/Controllers/MembersController.php [hunk 1]
```diff
@@ -3,8 +3,38 @@
 namespace App\Http\Controllers;
 
 use Illuminate\Http\Request;
+use App\Model\Member;
 
 class MembersController extends Controller
 {
-    //
+	protected $member;
```
Comment (4c0n) [inline app/Http/Controllers/MembersController.php]: Indentation does not match the methods below. Please correct :)

File: app/Http/Controllers/MembersController.php [hunk 2]
```diff
@@ -3,8 +3,38 @@
...
+
+	public function __construct(Member $member){
```
Comment (4c0n) [inline app/Http/Controllers/MembersController.php]: Please stick to the PSR-2 code style and run phpcs before you make a PR. (If you have questions on how to use phpcs, please ask me)

Comment (4c0n) [inline app/Http/Controllers/MembersController.php]: I like the dependency injection here.

File: public/js/lib/controller/MemberController.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
+var url = 'http://127.0.0.1:8000/dashboard/member/';
```
Comment (4c0n) [inline public/js/lib/controller/MemberController.js]: If possible use a relative url here.

File: resources/views/layouts/admin-layout.blade.php [hunk 4]
```diff
@@ -1,148 +0,0 @@
-<!doctype html>
-<html lang="en">
-<head>
-  <meta charset="utf-8" />
-  <link rel="icon" type="image/png" href="assets/img/favicon.ico">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
-
-  <title>Light Bootstrap Dashboard by Creative Tim</title>
```
Comment (4c0n) [inline resources/views/layouts/admin-layout.blade.php]: Thank you for not using this.

File: resources/views/layouts/dashboard-layout.blade.php [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
+    <meta name="description" content="">
+    <meta name="author" content="">
+    <link rel="icon" href="../../favicon.ico">
+
+    <title>Dashboard Template for Bootstrap</title>
```
Comment (4c0n) [inline resources/views/layouts/dashboard-layout.blade.php]: Did you make this template?

Comment (tomkelele) [inline resources/views/layouts/dashboard-layout.blade.php]: i used bootstrap dashboard demo for example , in header i just follow those HTML tags :D sorry for that.

**Pull request merged**